### ISSUE TITLE
Undo karma points

### ISSUE THREAD

**Issue by mariuscoto** [rosedu/wouso#768]
Teachers reported that they need an easy way to undo submitted karma points for a group.

**Comment by dosarudaniel** [comment]
They can give them -10*x gold points instead. (x is the number of karma points given earlier)

**Comment by gabrielivascu** [comment]
> They can give them -10*x gold points instead. (x is the number of karma points given earlier)

Yes, but doing this will create a new entry on the Activity page and that's not what we want.
Besides, the karma-points formula may be changed in the future.
